support-frontend/assets/helpers/abTests/abtestDefinitions.ts

```diff
@@ -140,4 +140,26 @@ export const tests: Tests = {
 			return !productParam || productParam === 'DigitalSubscription';
 		},
 	},
+	subscribeCheckoutImage: {
+		variants: [
+			{
+				id: 'control',
+			},
+			{
+				id: 'variant',
+			},
+		],
+		audiences: {
+			ALL: {
+				offset: 0,
+				size: 1,
+			},
+		},
+		isActive: false,
+		referrerControlled: false, // ab-test name not needed to be in paramURL
+		seed: 5,
+		targetPage:
+			'(/subscribe/paper/checkout$|/subscribe/weekly/checkout$|/subscribe/weekly/checkout/gift$)',
```

Comment:
Did we agree to only run this for GW so we don't overlap with the newspaper promo?

Reply:
Have refined this down now to `/subscribe/weekly/checkout$|/subscribe/weekly/checkout/gift$`, testing it now.